torchsynth/signal.py

```diff
@@ -20,3 +20,9 @@ def batch_size(self):
     def num_samples(self):
         assert self.ndim == 2
         return self.shape[1]
+
+    def new_empty(self, *args, **kwargs):
+        """
+        Implement [torch.Tensor.new_empty](https://pytorch.org/docs/stable/generated/torch.Tensor.new_empty.html) so that Signal can be [`deepcopy`](https://docs.python.org/3/library/copy.html#copy.deepcopy)ed.
```

Comment:
This comment is triggering lint on line-too-long

Reply:
It's fixed now